quix.yaml

```diff
@@ -96,7 +92,6 @@ deployments:
       cpu: 8000
       memory: 10240
       replicas: 1
-    libraryItemId: eec165e1-f648-4e58-9fd3-9af475a840e8
```

Comment:
I am suspicious this is a bug that Patrick and the team is working right now. Patrick can you double check we can merge this?

Reply:
@PatrickMiraP ?

Reply:
We detected an issue where we are losing the libraryItemId when you synchronize from one environment to the other. Emanuel is fixing that, but anyway this libraryItemId is only used now for statistics in our reports. Internally the deployment is always using now the Application related to the Deployment.

So I think it is safe to merge this for now. This will not happen in the near future with the fix Emanuel is preparing.